agr_indexer/src/main/java/org/alliancegenome/indexer/Main.java

```diff
@@ -48,7 +48,9 @@ public static void main(String[] args) {
 				Indexer i = (Indexer) ic.getIndexClazz().getDeclaredConstructor(IndexerConfig.class).newInstance(ic);
 				indexers.put(ic.getTypeName(), i);
 				if (ic.getRunInParallel()) {
-					parallelMap.put(ic.getTypeName(), i);
+/// TODO: stopped using parallel as it caused slowdown
+// parallelMap.put(ic.getTypeName(), i);
+					sequentialMap.put(ic.getTypeName(), i);
```

Comment:
instead of switching this i would suggest just setting the fixed workers to 1 this way at least the neo indexers and curation indexes can run in parallel